interface/ceed-qfunction.c

```diff
@@ -681,6 +617,70 @@ int CeedQFunctionAddOutput(CeedQFunction qf, const char *field_name,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Get the CeedQFunctionFields of a CeedQFunction
+
+  @param qf                  CeedQFunction
+  @param[out] input_fields   Variable to store input_fields
+  @param[out] output_fields  Variable to store output_fields
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Backend
```

Comment:
Perhaps Advanced or something suggesting that these are useful for higher level abstractions/libraries, but you probably don't need it when writing straight-line application code.